fix: Skeleton and arrow appears when login via concierge

nkdengineer opened the pull request:
<!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->

### Details
<!-- Explanation of the change or anything fishy that is going on -->

### Fixed Issues
<!---
1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing. For example, `$ https://github.com/Expensify/App/issues/<issueID>`.
2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).  For example, `PROPOSAL: https://github.com/Expensify/App/issues/<issueID>#issuecomment-1369752925`

Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
It MUST be an entire link to the github issue and your comment proposal ; otherwise, the linking and its automation will not work as expected.

Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):

$ https://github.com/Expensify/App/issues/<issueID>
$ https://github.com/Expensify/App/issues/<issueID(comment)>

Do NOT only link the issue number like this: $ #<issueID>
--->
$ https://github.com/Expensify/App/issues/39698
PROPOSAL: https://github.com/Expensify/App/issues/39698#issuecomment-2205488693


### Tests
<!---
Add a numbered list of manual tests you performed that validates your changes work on all platforms, and that there are no regressions present.
Add any additional test steps if test steps are unique to a particular platform.
Manual test steps should be written so that your reviewer can repeat and verify one or more expected outcomes in the development environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->
1. Navigate to /concierge
2. Sign-up with an account when app opens
3. Verify that: User should navigate to the concierge chat
- [x] Verify that no errors appear in the JS console

### Offline tests
<!---
Add any relevant steps that validate your changes work as expected in a variety of network states e.g. "offline", "spotty connection", "slow internet", etc. Manual test steps should be written so that your reviewer and QA testers can repeat and verify one or more expected outcomes. If you are unsure how the behavior should work ask for advice in the `#expensify-open-source` Slack channel.
--->

### QA Steps
<!---
Add a numbered list of manual tests that can be performed by our QA engineers on the staging environment to validate that your changes work on all platforms, and that there are no regressions present.
Add any additional QA steps if test steps are unique to a particular platform.
Manual test steps should be written so that the QA engineer can repeat and verify one or more expected outcomes in the staging environment.

For example:
1. Click on the text input to bring it into focus
3. Upload an image via copy paste
4. Verify a modal appears displaying a preview of that image
--->
1. Navigate to /concierge
2. Sign-up with an account when app opens
3. Verify that: User should navigate to the concierge chat
- [x] Verify that no errors appear in the JS console

### PR Author Checklist
<!--
This is a checklist for PR authors. Please make sure to complete all tasks and check them off once you do, or else your PR will not be merged!
-->

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/cf7b4119-a750-4f4a-b9f6-41268c75acb4


</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/43c81e07-235f-4180-9181-819f34e19771


</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/87ca17d7-44d2-4aa5-8171-bfd2bcf99b41


</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/618e71dd-9a35-4cd2-a4c3-3b73005604fe


</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/4a7e7fc2-792f-4378-bf46-3d07915651c4


</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/703390c8-0d47-4e4d-aac0-c853d097de14


</details>

nkdengineer committed:
fix: Skeleton and arrow appears when login via concierge

sobitneupane commented:
@nkdengineer Can you please upload the screen recordings for Android: Native and Android: mWeb Chrome. The issue is mainly reported on Android.

nkdengineer commented:
I'll update soon

nkdengineer commented:
@sobitneupane please check again

sobitneupane commented:
@nkdengineer Can you please merge main? When I try to navigate to `https://dev.new.expensify.com:8082/conceirge`, it's not here page is displayed. Can you reproduce it in your end?

https://github.com/user-attachments/assets/8ae392a1-ffdb-4fdc-acec-81f7df36264d

nkdengineer committed:
Merge branch 'main' into fix/39698

nkdengineer commented:
@sobitneupane I think you should turn off strict mode [here](https://github.com/Expensify/App/blob/750c34eca1531a48ace4fd5aca24caf3398d9176/src/CONFIG.ts#L99) and test again

https://github.com/user-attachments/assets/d77d7cb4-adea-4528-84d9-c45599d38458

sobitneupane reviewed (commented):
In native app (both in Android and IOS), the header is not shown before loading the concierge page.

<img width="447" alt="Screenshot 2024-07-26 at 19 22 39" src="https://github.com/user-attachments/assets/4d1eb994-3cd6-4668-94cf-70d69e1bca79">

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/user-attachments/assets/c94a33c8-55e0-4fcc-a8ba-9429926d3120

</details>


<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/c36552da-dd7a-4443-a08d-a6ea0fd47e18

</details>

<details>
<summary>iOS: mWeb Safari</summary>


https://github.com/user-attachments/assets/25972c4a-07c3-4253-a643-f73261c0fe39


</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/7b7715a2-6e73-419d-9aa7-acda88f8da5d

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

nkdengineer commented:
@sobitneupane I think this [bug](https://github.com/Expensify/App/pull/45460#pullrequestreview-2201855174) is expected, you can check more [here](https://github.com/nkdengineer/App/blob/98d8a5ae4bb52539669f79e007246417eb8c8f1e/src/pages/home/ReportScreen.tsx#L821-L824)

sobitneupane commented:
Sorry @nkdengineer But I don't understand how is it expected based on [this](https://github.com/nkdengineer/App/blob/98d8a5ae4bb52539669f79e007246417eb8c8f1e/src/pages/home/ReportScreen.tsx#L821-L824). Is [this](https://github.com/Expensify/App/pull/45460#pullrequestreview-2201855174) reproducible for reports other than Concierge?

nkdengineer commented:
Let me check again if it's also reproducible on the latest main.

nkdengineer committed:
Merge branch 'main' into fix/39698

nkdengineer committed:
add header skeleton view

nkdengineer commented:
@sobitneupane i added header skeleton view, please check again

sobitneupane commented:
@nkdengineer As you can notice in the following video, height of the skeleton header and length of rows in the skeleton header changes while the concierge is being loaded. Let's try to get this merged asap. Also, please make sure to test it in your end before making it ready for review.


https://github.com/user-attachments/assets/22db5dc0-2406-4ab7-b23a-d59e0d5e1966

nkdengineer commented:
@sobitneupane I'll update soon

nkdengineer committed:
Merge branch 'main' into fix/39698

nkdengineer committed:
fix: logic show header skeleton

nkdengineer commented:
@sobitneupane i updated, please check again

sobitneupane commented:
@nkdengineer I can still reproduce https://github.com/Expensify/App/pull/45460#issuecomment-2271474446 issue and https://github.com/Expensify/App/pull/45460#pullrequestreview-2201855174 issue.

https://github.com/user-attachments/assets/47ee3141-1a21-4bf4-aa02-774390d58282

sobitneupane commented:
@nkdengineer This is already a month old issue. Let's prioritize this PR. I am hoping to get this merged within next few days.

nkdengineer commented:
@sobitneupane I'm checking this now.

nkdengineer committed:
revert previous commit

nkdengineer committed:
Merge branch 'main' into fix/39698

nkdengineer committed:
remove withOnyx in header view

nkdengineer commented:
@sobitneupane After investigating more time, I see that the problem is the `ReportActionsSkeletonView` is rendered before `HeaderView` in the concierge chat then it takes the full height of the `ReportScreen` (Tested this by logging a `useEffect` in these components). The bug may be related to `withOnyx` in `HeaderView` since it's an old HOC then I removed this and the bug is fixed now.

sobitneupane commented on src/pages/ConciergePage.tsx:
```diff
@@ -56,7 +56,7 @@ function ConciergePage({session}: ConciergePageProps) {
     return (
         <ScreenWrapper testID={ConciergePage.displayName}>
-            <View style={[styles.borderBottom]}>
+            <View style={[styles.borderBottom, styles.appContentHeader]}>
```
@nkdengineer After the inclusion of `styles.appContentHeader`, the height of the 2nd skeleton Header is smaller than the first skeleton Header and actual Header.

https://github.com/user-attachments/assets/ef892ca7-e5da-4c0b-bdbc-d2d461bc9300


https://github.com/user-attachments/assets/b90e9a3a-bd10-42e3-aa8f-6fd3c4b4baba

nkdengineer replied:
@sobitneupane I updated to fix this case.

nkdengineer committed:
Merge branch 'main' into fix/39698

nkdengineer committed:
fix header skeleton width

nkdengineer committed:
fix crash bug

sobitneupane commented:
@nkdengineer Still reproducible. Height of skeleton header and actual header is different. We should maintain the existing styling (i.e, header styling in main brach). In the staging, the issue is not reproducible.


https://github.com/user-attachments/assets/74427b39-b9c2-4e3e-aac2-f0f9d3717f6b

Please make sure to test it in your end before making it ready for review.

nkdengineer committed:
Merge branch 'main' into fix/39698

nkdengineer committed:
fix height of skeleton in the large screen

nkdengineer commented:
> After the inclusion of styles.appContentHeader, the height of the 2nd skeleton Header is smaller than the first skeleton Header and actual Header.

> Height of skeleton header and actual header is different. I don't think it's a good idea to make change in actual header styling in HeaderView

@sobitneupane  Yesterday when I saw the video, I saw the bug was that the width of skeleton line was different and I understand it's the bug that you mentioned then I updated to fix this case and I also tested this case.

> Please make sure to test it in your end before making it ready for review.

I updated to fix the height of skeleton of concierge page.

nkdengineer committed:
update to use the policy hook

sobitneupane reviewed (commented):
### Screenshots/Videos
<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

https://github.com/user-attachments/assets/e8f0c1e1-2c37-4c7c-a8ac-b0f8037abef9


</details>

<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/60e156ca-7d76-4a50-83ea-1762151df504


https://github.com/user-attachments/assets/d163f030-d8cd-4ba1-9d8a-e483bd434068


<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/user-attachments/assets/7fc6fd39-202b-4842-b831-3f78106826b3

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/3a90f35b-abb9-4166-884f-cd4f22f5add5

https://github.com/user-attachments/assets/f52b5bf5-5e0c-4113-860f-16ff589e6390

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/c41f5144-014a-4d41-9360-ecc8acf24997

https://github.com/user-attachments/assets/d2364967-cd52-47e1-a926-f6a05b215d98

</details>

sobitneupane reviewed (approved):
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

yuwenmemon commented on src/pages/home/HeaderView.tsx:
```diff
@@ -62,9 +52,13 @@ type HeaderViewProps = HeaderViewOnyxProps & {
+function HeaderView({report, parentReportAction, reportID, onNavigationMenuButtonClicked, shouldUseNarrowLayout = false}: HeaderViewProps) {
     const [isDeleteTaskConfirmModalVisible, setIsDeleteTaskConfirmModalVisible] = React.useState(false);
     const [invoiceReceiverPolicy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${report.invoiceReceiver && 'policyID' in report.invoiceReceiver ? report.invoiceReceiver.policyID : -1}`);
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```
NAB: Add a note explaining this is because the reportIDs can be 0.

OSBotify commented:
:hand: This PR was not deployed to staging yet because QA is ongoing. It will be automatically deployed to staging after the next production release.

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/10408038339) to staging by https://github.com/yuwenmemon in version: 9.0.21-0 🚀

platform | result
---|---
🤖 android 🤖|success ✅
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅
🕸 web 🕸|success ✅

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/10456490628) to production by https://github.com/Beamanator in version: 9.0.21-4 🚀

platform | result
---|---
🤖 android 🤖|success ✅
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅
🕸 web 🕸|success ✅
